src/components/BackedBy/index.tsx

```diff
@@ -16,16 +48,18 @@ export const BackedBy = () => {
           marginBottom: 4,
         }}
       >
-        {Array.from({ length: 10 }).map((v, i) => {
+        {ORGS.map((org, i) => {
           return (
-            <Box
-              sx={{
-                backgroundColor: '#D9D9D9',
-                height: 150,
-                flex: '1 1 20%',
-              }}
-              key={i}
-            />
+            <ContainerBox key={i}>
+              <a href={org.url}>
+                <Image
+                  src={org.image}
+                  alt="Ilustration"
```

Comment:
I think the alt could be also configurable on the ORGS object, like alt: "ETHLatam" or "Kleros"